apps/arena/lib/arena/game/skill.ex

```diff
@@ -288,8 +288,8 @@ defmodule Arena.Game.Skill do
     Enum.reduce(effect.effect_mechanics, player, fn {mechanic_name, mechanic_params} = mechanic, player ->
       should_re_apply? =
         is_nil(Map.get(mechanic_params, :last_application_time)) or
-          now - Map.get(mechanic_params, :last_application_time) >=
-            mechanic_params.effect_delay_ms
+          (not is_nil(Map.get(mechanic_params, :effect_delay_ms)) and
```

Comment:
What is this new check for?

Reply:
Not all mechanics need to be reapplied so no `effect_delay_ms`